tests/__fixtures__/core/search-service.ts

```diff
@@ -11,7 +11,14 @@ const mockedUniversalResponse: UniversalSearchResponse = {
 const mockedVerticalResponse: VerticalSearchResponse = {
   uuid: '123',
   queryId: 'query-id',
-  verticalResults: null
+  verticalResults: {
+    appliedQueryFilters: [],
+    queryDurationMillis: 500,
+    verticalKey: 'vertical',
+    results: [],
+    resultsCount: 0,
+    source: 'KNOWLEDGE_MANAGER' as Source
```

Comment:
can we do `Source.KnowledgeManager` instead?

Reply:
that fixes the TS error but when running the Jest tests, all of the test suites that use vertical search fail with the error that it's referencing the `KnowledgeManager` property of `Source`, which is undefined. this is probably because we're mocking Core, so it can't find `Source`. I could re-define the `Source` enum in this file to fix both errors, but that seemed a little weird. I don't have a strong preference though if that seems better than the assertion

Reply:
I see, that's fine then